lib/screens/surveydetails/widget/question_paging_widget.dart

```diff
@@ -173,6 +176,15 @@ class _QuestionPagingWidgetState extends State<QuestionPagingWidget> {
     );
   }
 
+  Widget _buildNpsRatingWidget(QuestionUiModel question) {
+    return AnswerNpsWidget(
+      count: question.answers.length,
+      onRated: (int rating) {
```

Comment:
Just a small naming issue here. What do you think about `onRatingChange`?

Reply:
Nice, updated in [08382e4](https://github.com/nimblehq/flutter-ic-lydia-ryan/pull/93/commits/08382e4ebd4f8ad90876402f3ec300e8390007d2)